logback-classic/src/test/java/ch/qos/logback/classic/spi/ThrowableProxyTest.java

```diff
@@ -112,6 +112,24 @@ public void suppressedWithCause() throws InvocationTargetException, IllegalAcces
   }
 
   @Test
+  public void suppressedCyclic() throws InvocationTargetException, IllegalAccessException
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/tony19/logback/file/2371260924/issues/source?bid=3222618&fileBranchId=3222662#l115)